Java/src/test/java/AirportTest.java

```diff
@@ -74,27 +73,22 @@ public void test3() {
     }
 
     @Test
-    public void testHasAtLeastOneBomberInMilitaryPlanes() {
+    public void hasAtLeastOneBomberInMilitaryPlanes() {
```

Comment:
This is not a valid test. 
You have an input parameter - list of planes,
so expected particular quantity of planes, and validate that their type is valid.